Host/Mdis_Test/Mdis_Test.sh

```
@@ -320,7 +320,7 @@ function cleanOnExit() {
         # Kill process
         echo "${LogPrefix} kill process ${MdisTestBackgroundPID}"
 
-        if ! kill ${MdisTestBackgroundPID}
+        if ! kill ${MdisTestBackgroundPID} 2>/dev/null
```

Is this just making this error message get ignored?

---

Yes, this change is for only hiding the possible error logs from kill command.